Source/SourceKittenFramework/Request.swift

```diff
@@ -340,3 +340,70 @@ extension Request: CustomStringConvertible {
     /// A textual representation of `Request`.
     public var description: String { return String(UTF8String: sourcekitd_request_description_copy(sourcekitObject))! }
 }
+
+private func interfaceForModule(module: String) -> [String: SourceKitRepresentable] {
+    let xctestConfigurationPath = NSProcessInfo.processInfo().environment["XCTestConfigurationFilePath"]!
+    let buildPath = xctestConfigurationPath.substringToIndex(xctestConfigurationPath.rangeOfString("/Intermediates/")!.startIndex)
```

Comment:
Do the tests pass when they're run from `SourceKitten.xcworkspace` at least? Do you know of any other way to get the derived data directory for SourceKittenFramework? Or another way to get all the necessary compiler arguments to SourceKit for interface generation?